src/definitions/modules/progress.js

```diff
@@ -70,9 +70,9 @@ $.fn.progress = function(parameters) {
       module = {
         helper: {
           sum: function (nums) {
-            return nums.reduce(function (left, right) {
+            return Array.isArray(nums) ? nums.reduce(function (left, right) {
```

Comment:
While investigating this issue, `nums` are sometimes `undefined`,.
So I added safeguard for such situation.

Reply:
👍